config/notifiers.go

```diff
@@ -351,3 +360,28 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+type AWSSNSConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	AWSRegion string `yaml:"aws_region"`
+	TopicArn  string `yaml:"topicarn"`
```

Comment:
Hmm... usually unless we are forced to keep the nonstandard naming (like with protobuf fields), I think we'd still go with the proper Go naming style in the part of the code that we control (`TopicARN`, https://github.com/golang/go/wiki/CodeReviewComments#initialisms).